packages/labs/virtualizer/package.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "@lit-labs/virtualizer",
+  "version": "0.7.0",
+  "description": "Viewport-based virtualization (virtual scrolling) for Lit",
+  "author": "Google LLC",
+  "license": "BSD-3-Clause",
+  "main": "lit-virtualizer.js",
+  "module": "lit-virtualizer.js",
+  "type": "module",
+  "exports": {
+    ".": {
+      "default": "./lit-virtualizer.js"
+    },
+    "./layouts/flexWrap.js": {
+      "default": "./layouts/flexWrap.js"
+    },
+    "./layouts/flow.js": {
+      "default": "./layouts/flow.js"
+    },
+    "./layouts/grid.js": {
+      "default": "./layouts/grid.js"
+    },
+    "./LitVirtualizer.js": {
+      "default": "./LitVirtualizer.js"
+    },
+    "./virtualize.js": {
+      "default": "./virtualize.js"
+    }
+  },
+  "scripts": {
+    "bench": "tach --root=../.. --browser=chrome-headless test/benchmarks/basic.html --measure=fcp",
+    "bench:debug": "tach --root=../.. test/benchmarks/basic.html --measure=fcp",
+    "bench:scroll": "tach --force-clean-npm-install --config=test/benchmarks/scrollingBenchmarks.json",
+    "build": "wireit",
+    "build:copy-polyfill": "wireit",
+    "build:ts": "wireit",
+    "checksize": "rollup -c; rm lit-virtualizer.bundled.js lit-virtualizer-with-polyfills.bundled.js",
```

Comment:
Ideally these scripts would use wireit too so that they can ensure a fresh build

Reply:
Agreed.  I filed #3035 to update wireit.  I did a little in this PR but when I ventured further I shot myself in the foot-- would prefer to handle remainder in follow-up PR.